spread/project.go

```diff
@@ -124,6 +124,9 @@ type System struct {
 
 	// Only for Google so far.
 	SecureBoot bool `yaml:"secure-boot"`
+	// Select the minimum CPU platform on Google GCE, e.g. "Intel Skylake"
+	// See https://cloud.google.com/compute/docs/instances/specify-min-cpu-platform#availablezones
+	MinCpuPlatform string `yaml:"min-cpu-platform"`
```

Comment:
Feedback from Gustavo: 
- let's use  "CpuFamily" as the name so that it's generic and could also be used for e.g. linode
- map that then in the google backend to the `min-cpu-platform` they are using

Reply:
This is addressed now.